src/ui/components/SecondaryToolbox/SecondaryToolbox.css

```diff
@@ -26,6 +26,7 @@
   cursor: pointer;
   transition: color 200ms;
   display: flex;
+  flex: 0 0 auto;
```

Comment:
Buttons should fit their content (without growing or shrinking).

Reply:
I never know if basis should be auto or content. Is there a rule of thumb where you use content over auto?

Reply:
TBH I'm unclear on that as well. I've always used "auto".